### DISCUSSION TITLE
Add resourcePool as config item on IPI install on vSphere

### DISCUSSION THREAD

**Discussion by guliaka** [okd-project/okd#1103] [Q&A] [answered]
Folks,

Is it possible to add resourcePool as config item on IPI install on vSphere?? Because as is it puts newly provisioned cluster into root.

Thank you

**Comment by vrutkovs** [comment, marked as answer]
See https://issues.redhat.com/browse/CORS-1500 and https://github.com/openshift/installer/pull/5136
